src/components/v5/common/ActionSidebar/ActionSidebar.tsx

```diff
@@ -37,8 +38,15 @@ const ActionSidebar: FC<PropsWithChildren<ActionSidebarProps>> = ({
     ],
     cancelModalToggle: [isCancelModalOpen, { toggleOff: toggleCancelModalOff }],
   } = useActionSidebarContext();
-  const [isSidebarFullscreen, { toggle: toggleIsSidebarFullscreen }] =
+  const [isSidebarFullscreen, { toggle: toggleIsSidebarFullscreen, toggleOn }] =
     useToggle();
+
+  useLayoutEffect(() => {
+    if (localStorage.getItem(isFullScreen) === 'true') {
+      toggleOn();
+    }
+  }, [toggleOn]);
```

Comment:
It feels like if `useFullScreen` hook became a global one, this could use its return value instead of reading from local storage

Reply:
It is currently a page specific hook since `toggleFullScreenMode` refers specifically to the display preferneces toggle, so don't think we should use it globally.

I could separate the following into its own hook, but it seems worse than just accessing local storage directly:

```
const [isFullScreenMode, setIsFullScreenMode] = useState(false);

useLayoutEffect(() => {
    setIsFullScreenMode(localStorage.getItem(isFullScreen) === 'true');
  }, []);
  
return { isFullScreenMode, setIsFullScreenMode };

```

And would still need the conditional to call `toggleOn`, so not obviously reducing loc either.

Reply:
I'm still thinking that you could just copy and paste it into `hooks` outright as it is, the action sidebar doesn't have to use `toggleFullScreenMode` but just read the current setting value. In my eyes you set it in the user preferences page, but it's a global preference